import_models_to_verta/mlflow_databricks_bulk_import_example.ipynb

```diff
@@ -0,0 +1,341 @@
+    "                name=str(version.version),\n",
+    "                attrs={\n",
+    "                    \"er_id\": version.run_id,\n",
+    "                    \"mlflow_source\": version.source,\n",
+    "                    \"mlflow_user_id\": version.user_id,\n",
+    "                    \"mlflow_run_link\": version.run_link,\n",
+    "                    \"mlflow_creation_time\": version.creation_timestamp,\n",
+    "                    \"mlflow_last_updated_time\": version.last_updated_timestamp,\n",
+    "                    \"mlflow_status\": version.status,\n",
+    "                    \"mlflow_current_stage\": version.current_stage,\n",
+    "                    \"mlflow_tags\": version.tags,\n",
+    "                },\n",
+    "                labels=[\"mlflow_import\"],\n",
+    "            )\n",
+    "            model_messages[rm].append(f\"successfully created version {version.version}\")\n",
+    "        except Exception as err:\n",
+    "            model_messages[rm].append(f\"failed to create model version in Verta for {rm.name} - version: {version.version} due to {err}\")\n",
+    "            continue\n",
+    "\n",
+    "        # download artifacts\n",
+    "        try:\n",
+    "            outpath = mlflow.artifacts.download_artifacts(run_id=version.run_id)\n",
+    "        except Exception as err:\n",
+    "            model_messages[rm].append(f\"unable to download artifacts from {rm.name} - version run id; {version.run_id}\")\n",
+    "            outpath = None\n",
+    "\n",
+    "        if outpath:\n",
+    "            for file_name in os.listdir(outpath):\n",
+    "                if file_name not in protected_file_names:\n",
+    "                    verta_version.log_artifact(file_name, os.path.join(outpath, file_name))\n",
+    "                    model_messages[rm].append(f\"artifact logged in Verta: {file_name}\")\n",
+    "                else:\n",
+    "                    model_messages[rm].append(f\"cannot upload artifact {file_name} for {rm.name} as the file name conflicts with a protected file name in Verta\")\n",
+    "        else:\n",
+    "            model_messages[rm].append(f\"no artifacts found for {rm.name} - {version.version}\")\n",
+    "\n",
+    "        # set stage for version\n",
+    "        stage_error_message = f\"unable to set stage in Verta for {rm.name} - version: {version}, current_stage: {version.current_stage}\"\n",
+    "        try:\n",
+    "            if version.current_stage != \"None\":\n",
+    "                if version.current_stage == \"Staging\":\n",
+    "                    verta_version.change_stage(Staging())\n",
+    "                if version.current_stage == \"Production\":\n",
+    "                    verta_version.change_stage(Production())\n",
+    "                if version.current_stage == \"Archived\":\n",
+    "                    verta_version.change_stage(Archived())\n",
```

Comment:
Should we try reporting an error if `version.current_stage` is some other, unsupported value?